cron/index.js

```diff
@@ -17,7 +17,8 @@ const INTERVAL = process.env.INTERVAL || '0 5 * * *'
 // "0,30 * * * *" = "Every 30 minutes (at X:00 and X:30)"
 const FAILURE_INTERVAL = '0,30 * * * *'
 
-const PERIOD = Period(process.env.PERIOD)
+const currentPeriod = Period.fromString(process.env.PERIOD)
```

Comment:
It is documented in the file [`.env.in`](https://github.com/KTH/lms-antagna/blob/master/.env.in)… Which I forgot to update! So that's something to do yeah.

Are we fine with updating the `.env.in` and refering it from this `cron/index.js` file?

Reply:
Yes, that sounds like a good plan!